ttnn/cpp/pybind11/pytensor.cpp

```diff
@@ -1555,6 +1581,20 @@ void pytensor_module(py::module& m_tensor) {
             py::return_value_policy::reference)
         .def(
             "to_torch",
+            [](const Tensor& self) -> py::object { return detail::convert_tt_tensor_to_torch_tensor(self, true); },
+            R"doc(
+            Convert tensor to torch tensor using legacy padded shape.
```

Comment:
`Converts`

Reply:
This follows third person imperative style used by rest of documentation.

Reply:
Ah, in this file you are right, this isn't consistent across the codebase https://docs.tenstorrent.com/tt-metal/latest/ttnn/ttnn/api.html. Low priority, but ideally we should agree upon and standardize this.

Reply:
yea :D